packages/DropdownMenu/src/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./DropDownMenu";
```

Comment:
The second `D` should not be capitalized here.  Unless you want to use `DropDownMenu` consistently everywhere (I'd support that adjustment, dropdown isn't really a single word).

Reply:
yeah thats why I did it, as we are calling it `drop-down-menu` in the repo

Reply:
Dropdown is one word though..  :(

Reply:
Never mind, Oscar is correct. It’s one word. We should keep it like it was.

Reply:
Oh, yeah thats how I originally had it @mikrotron. Ok i'll change it back to that.

Reply:
I'll be reverting the paprika package name to `@paprika/dropdown-menu` againd